src/App.js

```diff
@@ -8,16 +8,16 @@ function App() {
         <img
           className="max-h-screen basis-2/5 object-cover"
           src={Background}
-          alt="SpaceBg"
+          alt="Universe Background"
```

Comment:
Not a blocker, but the idea of the alt is that people using screen readers (blind, poor of sight, etc) can understand what's going on. A descriptive alt would be better, maybe "A background image representing the Universe".

Reply:
Thanks! Didn't know, going to take that in consideration next time!